app/models/user.rb

```diff
@@ -5,4 +5,13 @@ class User < ApplicationRecord
   devise :database_authenticatable, :recoverable,
          :rememberable, :trackable, :validatable
   has_and_belongs_to_many :projects
+  ROLE_ADMIN = 1
+
+  def as_json(_options = {})
+    super(methods: [:admin])
+  end
+
+  def admin
```

Comment:
Isn't ruby convention to end predicates with '?'

Reply:
Cant because i am exporting it to json